src/components/KonnectorEdit.jsx

```diff
@@ -81,6 +81,9 @@ export const KonnectorEdit = ({
   if (fields.accountName)
     fields.accountName.placeholder = getAccountName(account)
 
+  if (account.oauth)
+    account.auth = Object.assign({}, account.auth, account.oauth)
```

Comment:
I suggest to use spread operator to be consistent `account.auth = { ...account.auth, ...account.oauth }`. What do you think ?

Reply:
Also, I am not a big fan of mutating `account.auth`. Do you think it's possible to use a kind of `auth` variable ?

Reply:
Since we pass the account object everywhere that will make things more complicated I think :/